2021-07-01 15:59:52  [ main:0 ] - [ ERROR ]  error on using scnnar
java.util.InputMismatchException
	at java.util.Scanner.throwFor(Unknown Source)
	at java.util.Scanner.next(Unknown Source)
	at java.util.Scanner.nextInt(Unknown Source)
	at java.util.Scanner.nextInt(Unknown Source)
	at main.java.Main.main(Main.java:16)
2021-07-01 16:06:35  [ main:0 ] - [ ERROR ]  error on using scnnar
java.util.InputMismatchException
	at java.util.Scanner.throwFor(Unknown Source)
	at java.util.Scanner.next(Unknown Source)
	at java.util.Scanner.nextInt(Unknown Source)
	at java.util.Scanner.nextInt(Unknown Source)
	at main.java.Main.main(Main.java:19)
